aws-configure-credentials-oidc/orb_version.txt

```diff
@@ -0,0 +1 @@
+ovotech/aws-configure-credentials-oidc@0.1.0
```

Comment:
I believe circleCI overwrites the version to 1.0.0 on the first publish, if it's 0.x.x (I couldn't find any docs stating that, but it looked like that happened to my last new orb)

Reply:
Thanks. Updated to 1.0.0 to be on the safe side